packages/core/src/components/accordion/accordion.tsx

```diff
@@ -12,12 +12,17 @@ export class TdsAccordion {
   /** Set the variant of the Accordion. */
   @Prop() modeVariant: 'primary' | 'secondary' = null;
 
+  /** Removes the bottom border of the last Accordion item. */
+  @Prop() noLastBorder: boolean = false;
```

Comment:
@theJohnnyMe do we have any other examples of this sort of prop, can't think of any from the top of my head? Just thinking for naming alignment.

Reply:
Maybe, just go with `hideLastBorder`, as we use hide a lot?

Reply:
I have seen a name like `noIcon` in the message component. But of course I can change it if you want to.

Reply:
Now I'm using `hideLastBorder`.